### DISCUSSION TITLE
Welcome to EFCore.Visualizer Discussions!

### DISCUSSION THREAD

**Discussion by Giorgi** [Giorgi/EFCore.Visualizer#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GeorgiGeorgievGitHub** [comment]
Hi,
Can you tell me if your extension works on Visual Studio 2026 community Version: 18.2.1?

**Reply by Giorgi** [reply]
I haven't tried that exact version but it should work.

**Reply by GeorgiGeorgievGitHub** [reply]
For the first time i need to work with Entity Framework till now in our company always use Dapper. That's why i send you a photo where you can see in debug what happens.
<img width="1682" height="301" alt="Screenshot 2026-01-30 091505" src="https://github.com/user-attachments/assets/a5bd78fc-9902-4968-bdba-5afc6f8b8d02" />
Variable "allContracts" is IQueryable
